src/contract-access-controller.js

```diff
@@ -2,43 +2,57 @@
 
 const AccessController = require('./access-controller-interface')
 
-const type = 'eth-contract/my-contract-ac'
+const type = 'eth-contract'
```

Comment:
As discussed elsewhere, these should have unique tags instead of just `eth-contract` to allow multiple different contract acs to be used at the same time.